providers/hatom-htm.ts

```diff
@@ -0,0 +1,83 @@
+import { LockedEgldProvider } from '@libs/common';
+import { Injectable } from '@nestjs/common';
+import { LiquidStakingUtils } from 'libs/common/src/utils/liquid-staking.utils';
+import BigNumber from 'bignumber.js';
+
+@Injectable()
+export class HatomHTMProvider extends LockedEgldProvider {
```

Comment:
why do we need 2 providers? I suggest that we only have one provider that will compute everything that is needed for each address.